pkg/operator/periodicbackupcontroller/periodicbackupcontroller_test.go

```diff
@@ -114,6 +116,7 @@ func TestSyncLoopFailsDegradesOperator(t *testing.T) {
 		backupsClient:         operatorFake.ConfigV1alpha1(),
 		kubeClient:            client,
 		operatorImagePullSpec: "pullspec-image",
+		backupVarGetter:       backuphelpers.NewDisabledBackupConfig(),
```

Comment:
I think we also need to add a controller level test here that shows you:
* by default have no backup set ("NewDisabledBackupConfig")
* when there is a default CRD, you set it in the backupVarGetter
* when you remove the CRD again it's back to "no backup set" like the first case

Reply:
created this test https://github.com/openshift/cluster-etcd-operator/pull/1305/commits/f21b98a8afc441a3b9ebb8a14436c9c76c697124